README.md

```diff
@@ -78,12 +78,21 @@ Example:
 let _t = true;
 let _f = false;
 ```
-#### Boolean Comparison Operators
+#### Comparison Operators
 
 | Symbol |  Meaning  | Example |
 |:-----|:--------:|------:|
 | >   | Greater than | `1 > 2`|
-| <   |  Less than  a| `2 < 1`|
+| <   |  Less than | `2 < 1`|
+
+#### Logical Operators
+
+| Symbol |  Meaning  | Example |
+|:-----|:--------:|------:|
+| ==   | Equal | `1 == 2`|
+| !=   | Not Equal | `2 < 1`|
```

Comment:
Example uses the wrong operator.

Reply:
Oops. fixed :point_right: 56de71d397e96b4b318e3ca6d79211f9b4d9efa8